template/_config.js

```diff
@@ -0,0 +1,6 @@
+System.config({
```

Comment:
If it is supporting a js export, maybe we can somehow read the list of available aliases from nuxt?

Reply:
Could you be more specific with "available aliases from nuxt". What do you mean?